nzshm_common/grids/region_grid.py

```diff
@@ -0,0 +1,24 @@
+from enum import Enum
+
+from nzshm_common.grids.nz_0_2_nb_1 import NZ_0_2_nb_1
+from nzshm_common.grids.wlg_0_01_nb_1 import WLG_0_01_nb_1
+from nzshm_common.grids.wlg_0_05_nb_1 import WLG_0_05_nb_1
+
+
+class RegionGrid(Enum):
+    NZ_0_2_NB_1 = ('NZ', 0.2, 1, NZ_0_2_nb_1())
```

Comment:
Oakley maybe you can use a namedtuple here t(or simlar) t give those enum values some more meaning,.

Reply:
e.f. I'm guessing that the 3rd item 1 means version 1?? but I can't be sure. It'd be nice if that was obvious 

some thing like
```
RegionGridEnumEntry = namedtuple("RegionGridEnumEntry name grid_resoution version file")
```
 might work in enum - I don't know?